pallets/stream-payment/src/lib.rs

```diff
@@ -361,12 +361,6 @@ pub mod pallet {
         },
     }
 
-    /// Freeze reason to use if needed.
```

Comment:
Since the actual lock/freeze implementation is done outside of the pallet, I left both FreezeReason and HoldReason if one need it.

Reply:
Ok, should I revert this change then?

Reply:
oh I see, that is interesting. Then we can leave it yes